Resources/config/odm.xml

```diff
@@ -16,6 +16,10 @@
         <parameter key="doctrine_phpcr.odm.cache.memcache_host">localhost</parameter>
         <parameter key="doctrine_phpcr.odm.cache.memcache_port">11211</parameter>
         <parameter key="doctrine_phpcr.odm.cache.memcache_instance.class">Memcache</parameter>
+        <parameter key="doctrine_phpcr.odm.cache.memcached.class">Doctrine\Common\Cache\MemcachedCache</parameter>
+        <parameter key="doctrine_phpcr.odm.cache.memcached_host">localhost</parameter>
+        <parameter key="doctrine_phpcr.odm.cache.memcached_port">11211</parameter>
```

Comment:
To be honest, I just added the parameters and this config magically worked : 

```
doctrine_phpcr:
    odm:
        metadata_cache_driver:
            type: memcached
```

Let me find where the magic happens.. ;)